packages/nimble-components/src/anchor-tabs/tests/anchor-tabs.mdx

```diff
@@ -0,0 +1,28 @@
+import {
+    Controls,
+    DocsStory,
+    Meta,
+    Stories,
+    Title,
+    Description
+} from '@storybook/blocks';
+import TargetDocs from '../../patterns/anchor/tests/target-docs.mdx';
+import * as anchorTabsStories from './anchor-tabs.stories';
+
+<Meta of={anchorTabsStories} />
+
+<Title of={anchorTabsStories} />
+<Description of={anchorTabsStories} />
+
+<DocsStory of={anchorTabsStories.anchorTabs} expanded={false} />
+<Controls of={anchorTabsStories.anchorTabs} />
+
+# Usage Docs
+
+## Accessibility
```

Comment:
Why is this Accessibility header here? I don't think the target docs are really an accessibility topic (more general principles of web design), and the following paragraph about Angular is definitely not accessibility. Not sure if we should separate that second paragraph under an Angular subheading, or just leave everything directly under Usage Docs.

Reply:
My thoughts were that routing correctness was part of making an app accessible, but I can see your point of view that it's really just overall usability rather than strictly an accessibility consideration. I don't have a strong preference. I'll move them out from under the _Accessibility_ header.